src/Microsoft.SqlTools.ServiceLayer/Localization/transXliff/sr.es.xlf

```diff
@@ -398,7 +398,7 @@
         <note/>
       </trans-unit>
       <trans-unit id="EE_ExecutionInfo_InitializingLoop">
-        <source>Starting execution loop of {0} times...</source>
+        <source>Beginning execution loop</source>
         <target state="new">Iniciando bucle de ejecución de {0} veces...</target>
```

Comment:
Again if we could alter to `Iniciando bucle de ejecución` this would be ideal. We no longer count number of times for this, so the translation would include `{0}` which looks wrong.